npm/angular/README.md

```diff
@@ -3,6 +3,11 @@
 Mount Angular components in the open source [Cypress.io](https://www.cypress.io/) test runner
 
 > **Note:** This package is bundled with the `cypress` package and should not need to be installed separately. See the [Angular Component Testing Docs](https://docs.cypress.io/guides/component-testing/angular/overview) for mounting Angular components. Installing and importing `mount` from `@cypress/angular` should only be done for advanced use-cases.
+
+## Requirements
+
+- Angular 17.2.0+
```

Comment:
Do we need a comment here about 13 as well?

Reply:
added once on rebase